doc/source/admin/framework_dependencies.rst

```diff
@@ -0,0 +1,573 @@
+    decorator-4.0. 100% |############################################| Time: 0:00:00  20.38 MB/s
+    docutils-0.12- 100% |############################################| Time: 0:00:00   2.21 MB/s
+    ecdsa-0.11-py2 100% |############################################| Time: 0:00:00 762.58 kB/s
+    markupsafe-0.2 100% |############################################| Time: 0:00:00 931.23 kB/s
+    mercurial-3.4. 100% |############################################| Time: 0:00:00   5.36 MB/s
+    nose-1.3.7-py2 100% |############################################| Time: 0:00:00   1.12 MB/s
+    paste-1.7.5.1- 100% |############################################| Time: 0:00:00   1.91 MB/s
+    pytz-2015.7-py 100% |############################################| Time: 0:00:00   1.08 MB/s
+    repoze.lru-0.6 100% |############################################| Time: 0:00:00 465.26 kB/s
+    requests-2.9.1 100% |############################################| Time: 0:00:00   2.28 MB/s
+    six-1.10.0-py2 100% |############################################| Time: 0:00:00 477.04 kB/s
+    sqlalchemy-1.0 100% |############################################| Time: 0:00:00   4.25 MB/s
+    sqlparse-0.1.1 100% |############################################| Time: 0:00:00 774.57 kB/s
+    webob-1.4.1-py 100% |############################################| Time: 0:00:00 819.13 kB/s
+    babel-2.1.1-py 100% |############################################| Time: 0:00:00   5.53 MB/s
+    bx-python-0.7. 100% |############################################| Time: 0:00:00   5.11 MB/s
+    mako-1.0.3-py2 100% |############################################| Time: 0:00:00 813.04 kB/s
+    paramiko-1.15. 100% |############################################| Time: 0:00:00   1.23 MB/s
+    pastedeploy-1. 100% |############################################| Time: 0:00:00 721.20 kB/s
+    requests-toolb 100% |############################################| Time: 0:00:00 856.06 kB/s
+    routes-2.2-py2 100% |############################################| Time: 0:00:00 666.70 kB/s
+    bioblend-0.7.0 100% |############################################| Time: 0:00:00   1.15 MB/s
+    fabric-1.10.2- 100% |############################################| Time: 0:00:00 843.81 kB/s
+    Extracting packages ...
+    [      COMPLETE      ]|###############################################################| 100%
+    Linking packages ...
+    [      COMPLETE      ]|###############################################################| 100%
+    #
+    # To activate this environment, use:
+    # $ source activate galaxy
+    #
+    # To deactivate this environment, use:
+    # $ source deactivate
+    #
+    $ source activate galaxy
+    discarding /home/nate/conda/bin from PATH
+    prepending /home/nate/conda/envs/galaxy/bin to PATH
+    $ pip install --index-url=https://wheels.galaxyproject.org/simple/ -r lib/galaxy/dependencies/requirements.txt
```

Comment:
No I'm going to do them. My queue this week has been getting longer and the priority of other things keeps getting higher.